apis/asset_registry.json

```diff
@@ -137,91 +186,116 @@
-              "application/json" : {
-                "schema" : {
-                  "type" : "object",
-                  "properties" : {
-                    "geoid" : {
-                      "type" : "string",
-                      "description" : "24 hexit string. The geo-identifier for an overlapping region"
-                    },
-                    "boundary" : {
-                      "type" : "string",
-                      "description" : "WKT string of the Polygon"
-                    },
-                    "meta-data" : {
-                      "type" : "object",
-                      "description" : "Dictionary of key-value pairs that are optionally posted by the creator (if they exist)"
+        "get" : {
+          "tags" : [ "authenticated" ],
+          "summary" : "Get an asset boundary with its geoid",
+          "operationId" : "getAsset",
+          "description" : "",
+          "parameters" : [ {
+            "in" : "query",
+            "name" : "geoid",
+            "required" : true,
+            "description" : "A 24 hexit string. The identifier for an overlapping region.",
+            "schema" : {
+              "type" : "string"
+            }
+          }, {
+            "in" : "query",
+            "name" : "includeS2Indices",
+            "required" : false,
+            "description" : "Include the S2 indices (list of Level 20, Level 19, Level 15. Level 8 and level 5) of all the indices that intersect the assetid\n",
+            "schema" : {
+              "type" : "integer",
+              "format" : "int32",
+              "minimum" : 0,
+              "maximum" : 1
+            }
+          }, {
+            "$ref" : "#/components/parameters/accessKey"
+          }, {
+            "$ref" : "#/components/parameters/signature"
+          } ],
+          "responses" : {
+            "200" : {
+              "description" : "GeoJSON payload with key-value pairs",
+              "content" : {
+                "application/json" : {
+                  "schema" : {
+                    "type" : "object",
+                    "properties" : {
+                      "geoid" : {
+                        "type" : "string",
+                        "description" : "24 hexit string. The geo-identifier for an overlapping region"
+                      },
+                      "boundary" : {
+                        "type" : "string",
+                        "description" : "WKT string of the Polygon"
+                      },
+                      "meta-data" : {
+                        "type" : "object",
+                        "description" : "Dictionary of key-value pairs that are optionally posted by the creator (if they exist)"
+                      }
                     }
                   }
                 }
               }
             }
           }
         }
-      }
-    },
-    "/findAssetForPoint" : {
-      "get" : {
+      },
+    "/fetch-fields-for-a-point" : {
+      "post" : {
```

Comment:
should this be `get` instead of `post`?

Reply:
@MHA10 @Aanish97

Reply:
@dockerymick as it's not just a simple GET request with only query parameters, the request body is also being sent in JSON format, which is why it's considered as a POST request.